src/main/java/org/codehaus/mojo/buildhelper/AbstractAddResourceMojo.java

```diff
@@ -47,10 +47,25 @@ public abstract class AbstractAddResourceMojo extends AbstractMojo {
     @Parameter(readonly = true, defaultValue = "${project}")
     private MavenProject project;
 
+    /**
+     * Skip plugin execution.
+     *
+     * @since 3.4.1
+     */
+    @Parameter(property = "skip", defaultValue = "false")
```

Comment:
I would like to consider to rename property to something like: `build-helper.skip`, property like `skip` can be to more general and we can have a conflict with other plugins.